LaunchScreenDemo/AppDelegate.swift

```diff
@@ -13,9 +13,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         
         //same navigation bar color for entire application
-        let appearance = UINavigationBarAppearance()
-        UINavigationBar.appearance().scrollEdgeAppearance = appearance
-        appearance.backgroundColor = .red
+//        let appearance = UINavigationBarAppearance()
```

Comment:
- The commented out code for setting the navigation bar appearance should be removed since it is not being used.
- The print statement can also be removed if it is not needed for debugging purposes.